src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
```

Comment:
add `display: block`
to avoid this 
![Screenshot 2023-03-22 at 19 30 59](https://user-images.githubusercontent.com/117587463/226989208-ffdb9e1c-6725-4915-b0b4-4457e15e9a38.png)